src/components/profile.js

```diff
@@ -32,38 +37,86 @@ class Profile extends Component {
 		return view;
 	}
 
+	setChanges( key ) {
+		return value => {
+			let changes = {};
+			changes[ key ] = value;
+
+			this.setState( changes );
+
+			return this.state.profile.save(
+				changes,
+				{ patch: true }
+			);
```

Comment:
You don't have to change this to land, but I'm now very wary of passing "smart" objects around React views. 

I'd rather you pass down a method so you can call `this.props.saveProfile(changes)` and the call to the model method `save` is done wherever that model is instantiated. Once you do that, `this.props.profiles` can be the serialized POJO model instead of the Ampersand object. The new `App` component of my Away PR shows this pattern.